.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
what is this ignoring

Reply:
husky creates a folder with name _. That folder contains scripts which run before/after git actions like commit

Reply:
I see, husky is weird lol